.github/workflows/smoke.yml

```diff
@@ -66,21 +64,15 @@ jobs:
         echo "GOPATH=$(go env GOPATH)" >> $GITHUB_ENV
         echo "$(go env GOPATH)/bin" >> $GITHUB_PATH
 
-    - name: Build snap
+    - name: Build local Juju
       shell: bash
       run: |
-        set -euxo pipefail
-        snapcraft --use-lxd
-
-    - name: Install snap
-      shell: bash
-      run: |
-        set -euxo pipefail
-        sudo snap install *.snap --dangerous --classic
+        make go-build
```

Comment:
Can I suggest a `make install` instead? That way we will only build the arch we are running on and we don't have to hard code paths or extend the PATH env.

Reply:
It would also allow us to smoke test building juju for different platforms as well.

Reply:
sounds good